src/pytest_workflow/schema/schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "title": "pytest-workflow YAML",
+  "description": "A YAML file describing a test for the pytest-workflow plugin",
+  "type": "object",
+  "properties": {
+    "arguments": {
+      "description": "arguments for the executable",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "results": {
+      "description": "A dictionary containing the results",
+      "type": "object",
+      "properties": {
+        "files": {
+          "description": "files produced by the workflow",
+          "type": "array",
+          "items": {
+            "type":"object",
+            "properties": {
+              "path": {
+                "type": "string"
+              },
+              "md5sum": {
+                "type": "string"
```

Comment:
You can set `"pattern": "^[a-f0-9]{32}$"`  here as well, to force an md5 pattern.

Reply:
Great suggestion!